includes/SVG.php

```diff
@@ -117,6 +92,16 @@ public function init() {
 		add_filter( 'site_option_upload_filetypes', [ $this, 'filter_list_of_allowed_filetypes' ] );
 	}
 
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+		$experiments = new Experiments();
+		return $experiments->is_experiment_enabled( 'enableSVG' );
```

Comment:
I'd recommend using constants for the experiments names, otherwise a simple typo will make something like the above behave erratically:
```suggestion
		return $experiments->is_experiment_enabled( Experiment::ENABLE_SVG );
```

Reply:
Not really related to the change in hand. I also disagree, as there are lots and lots of feature flags and that would become messy.

Reply:
@spacedmonkey Could you create a ticket for this suggestion? :-)